confluence/server/scio_search/src/main/java/com/askscio/atlassian_plugins/confluence/impl/ScioSearchServletFilter.java

```diff
@@ -69,7 +69,10 @@ public void doFilter(
     }
 
     final HttpServletRequest httpreq = (HttpServletRequest) servletRequest;
-    if (!httpreq.getRequestURI().contains("viewpage")
+    // Saving a page or blogpost fires: PUT http://bharath-test:8090/rest/api/content/65603?status=draft
```

Comment:
how are deletes handled? Did we verify this for creates also?

Reply:
should we make it resilient and send POST and DELETE events also? for example what happens when comments are added/deleted

Reply:
updated PR description with info about deletes. also sending POST & DELETE now. 

The one downside is -- DELETE of a comment triggers a /rest/api/content/1234 call where 1234 is id of the comment and we receive and process this event (which ends in errors since it is not a page). But this is probably good for newer confluence versions where pages also get DELETEs.